client/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -20,16 +20,30 @@
 tie.factory('PythonPrereqCheckService', [
   'PrereqCheckFailureObjectFactory', 'PREREQ_CHECK_TYPE_BAD_IMPORT',
   'PREREQ_CHECK_TYPE_MISSING_STARTER_CODE', 'SUPPORTED_PYTHON_LIBS',
+  'PREREQ_CHECK_TYPE_GLOBAL_CODE',
   function(
       PrereqCheckFailureObjectFactory, PREREQ_CHECK_TYPE_BAD_IMPORT,
-      PREREQ_CHECK_TYPE_MISSING_STARTER_CODE, SUPPORTED_PYTHON_LIBS) {
+      PREREQ_CHECK_TYPE_MISSING_STARTER_CODE, SUPPORTED_PYTHON_LIBS,
+      PREREQ_CHECK_TYPE_GLOBAL_CODE) {
 
     var rightTrim = function(str) {
       // Remove trailing white space at end of string.
       var RIGHT_TRIM_PATTERN = /\s+$/;
       return str.replace(RIGHT_TRIM_PATTERN, '');
     };
 
+    var checkGlobalCallsPresent = function(code) {
+      var codeLines = code.split('\n');
+      for (var i = 0; i < codeLines.length; i++) {
+        var line = rightTrim(codeLines[i]);
+        var pattern = /(^\s+)|(^def)|(^import)/;
```

Comment:
Can we rename pattern to something like VALID_LINE_PATTERN or something so that it's more clear what the regex matches?